src/imio/history/browser/views.py

```diff
@@ -86,6 +86,14 @@ def showColors(self):
         """
         return True
 
+    def showRevisionInfos(self):
+        """Return True if the type of the context is versioned. """
+        pr = getToolByName(self.context, 'portal_repository')
+        if self.context.getTypeInfo().getId() in pr.getVersionableContentTypes():
```

Comment:
@DieKatze 

portal_type is stored on the element, get it directly instead of getting the portal_type object then it's id...